src/crab/array_domain.cpp

```
@@ -419,11 +428,10 @@ std::optional<uint8_t> get_value_byte(const NumAbsDomain& inv, const offset_t o,
     return bytes[o % width];
 }
 
-std::optional<LinearExpression> ArrayDomain::load(StackCellRegistry& cells, const NumAbsDomain& inv,
-                                                  const DataKind kind, const Interval& i, const int width,
-                                                  const bool big_endian) {
+std::optional<LinearExpression> ArrayDomain::load(const NumAbsDomain& inv, const DataKind kind, const Interval& i,
+                                                  const int width, const bool big_endian) {
     if (const std::optional<Number> n = i.singleton()) {
-        offset_map_t& offset_map = cells.get(kind);
+        offset_map_t& offset_map = cells_.get_mutable().get(kind);
```

Cppcheck (2.20.0)

[style] The function 'fill_ones' is never used.

(unusedFunction)